lib/tasks/forms.rake

```diff
@@ -77,3 +97,23 @@ def summarise_form_documents
 
   { form_summary: { form_counts:, form_document_counts: } }
 end
+
+def set_submission_type(submission_type, form_id)
+  Rails.logger.info("setting submission_type to #{submission_type} for form: #{form_id}")
+
+  form = Form.find(form_id)
+  form.submission_type = submission_type
+  form.save!
+
+  made_live_form = form.made_live_forms.last
```

Comment:
Would `form.made_live_forms.last` return the old form snapshot if the form was archived? Do we want to update the archived form document?

Reply:
Oh very good point! We can use `form.live_version` instead I think

Reply:
Ah not quite actually because we need update it and that method returns the JSON - but we can check the form status